packages/fluxcd-source-controller/1.1.2/package.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: fluxcd-source-controller.community.tanzu.vmware.com.1.1.2
+spec:
+  refName: fluxcd-source-controller.community.tanzu.vmware.com
+  version: 1.1.2
+  releaseNotes: "Releasing TAP compatible source controller"
```

Comment:
We should update the release not here. This is not compatible with tap.

Reply:
Updated

Reply:
> I would suggest better to add this package to some dummy repo then install this package and check everything works as expected before merging.

That's done, by referring feature branch from cloned repo and creating packages & repo using VMware internal pipelines